package-lock.json

```diff
@@ -1,3155 +1,6 @@
 {
-  "name": "opendataday",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
can we back out the lockfile changes (I think you're running NPM 6 or lower and its rewritten the lockfile back to v1 format?)

Reply:
Argh sorry. Should not have done `git -a`. Should be good now